src/scrollMenu.spec.js

```diff
@@ -2219,12 +2219,12 @@ describe('functions', () => {
     it('should check visibility status on every window resize and change that status if need it', () => {
       const wrapper = mount(<ScrollMenu {...props} />);
 
-      expect(wrapper.state().rightArrowVisible).toEqual(false);
```

Comment:
Better keep existing test, rename it (it use `hideSingleArrow=true`) and add a new one for `hideSingleArrow=false`.
Like this you fix one issue and create another.